source/myukfast/my_company_details.md

```diff
@@ -0,0 +1,38 @@
+# My Company
+
+```eval_rst
+   .. title:: MyUKFast | My Company
+   .. meta::
+      :title: MyUKFast | Contacting my Account Manager | UKFast Documentation
+      :description: Page providing information about changing your company details inside MyUKFast
+      :keywords: ukfast, company name, company address, my company, VAT number
+```
+
+## Company Address Details
+When you move office or change mailing address, you can edit your address in the [My Company](https://my.ukfast.co.uk/account/view-edit.php) section of MyUKFast.
+
+ 1. Click **My Account >> Company Details**
+ 2. Your existing details will be displayed
+ 3. Edit fields as necessary
+ 4. All fields marked with * are required.
+ 5. **Save details**
+
+Within this page, it is also possible to change the **Primary Contact* (select from drop down menu) and choose between **Paperless** and **Postal** Invoicing.
+
```

Comment:
Drop the postal/paperless bit.. I don't think we do postal anymore and need to remove that option from the page